californium-core/src/main/java/org/eclipse/californium/core/network/stack/ReliabilityLayer.java

```diff
@@ -364,6 +365,15 @@ public void run() {
 					LOGGER.log(Level.FINE, "Timeout: retransmission limit reached, exchange failed, message: {0}", message);
 					exchange.setTimedOut();
 					message.setTimedOut(true);
+					OptionSet options = message.getOptions();
+					boolean isBlockwise = options.hasBlock1() || options.hasBlock2();
+					if (message instanceof Request && isBlockwise) {
```

Comment:
Great, thanks! What is the best approach to change it - to add a new PR or to add the next commit to this?